### PR TITLE
Remove hardcoding of passing in reviews to get business tags

### PR DESCRIPTION
**Pull request by graceehuu**
Issue(#33)

For the NLP API call to get entities, removed hard-coded example to pass in reviews as param
Made call of `getReviewsEntities(reviews)` synchronous
Removed button for debugging this feature

### CODE REVIEW

File: capstone/src/main/webapp/script.js [hunk 1]
```diff
@@ -326,25 +326,23 @@ function isStringEmpty(str) {
...
-    'Content-Type': 'application/json',
-  },
-};
-
-function getReviews() {
+async function getReviewsEntities(reviews) {
   // TODO(#33): integrate with actual reviews of businesses
-  const review = 'Really good pizza, nice wine, reasonable prices and great music.';
-  getEntities(review);
+  const reviewsEntities = await getEntities(reviews);
```
Comment (gannerr) [inline capstone/src/main/webapp/script.js:331]: is there maybe an issue you could attach to here?

File: capstone/src/main/webapp/script.js [hunk 2]
```diff
@@ -326,25 +326,23 @@ function isStringEmpty(str) {
...
 }
 
 /** send POST request to Cloud Natural Language API for entity recognition */
-function getEntities(message) {
-  const url = '/nlp-entity-recognition?message=' + message;
-  fetch(url, requestParamPOST).then((response) => response.json()).then((entities) => {
-    const businessTags = entities;
+function getEntities(messages) {
+  const url = '/nlp-entity-recognition?messages=' + messages;
+  const requestParamPOST = {
```
Comment (gannerr) [inline capstone/src/main/webapp/script.js:338]: what's the reasoning for moving the requestParamPOST to inside the function? what if in the future, other functions also want to call post requests?

File: capstone/src/main/webapp/script.js (hunk 1 above)
Comment (graceehuu) [reply capstone/src/main/webapp/script.js:331]: this is tied w issue#33 (see comment above)

File: capstone/src/main/webapp/script.js (hunk 2 above)
Comment (graceehuu) [reply capstone/src/main/webapp/script.js:338]: I'm not sure why, but when I tested it again I kept getting an error about how I had to initialize `requestParamPOST`, so I moved it to inside

File: capstone/src/main/webapp/script.js (hunk 1 above)
Comment (gannerr) [reply capstone/src/main/webapp/script.js:331]: ohhh very nice

File: capstone/src/main/webapp/script.js (hunk 2 above)
Comment (gannerr) [reply capstone/src/main/webapp/script.js:338]: bet

**Comment by terencedignon** [comment]
Hey, if possible resolve the linter errors. Otherwise LGTM :)

**Pull request merged**